test/jdk/TEST.groups

```diff
@@ -620,7 +620,9 @@ jdk_core_manual_no_input = \
     java/util/Vector/Bug8148174.java \
     com/sun/net/httpserver/simpleserver/CommandLinePortNotSpecifiedTest.java \
     com/sun/net/httpserver/simpleserver/jwebserver/CommandLinePortNotSpecifiedTest.java \
-    javax/xml/jaxp/datatype/8033980/GregorianCalAndDurSerDataUtil.java
+    javax/xml/jaxp/datatype/8033980/GregorianCalAndDurSerDataUtil.java \
+    java/foreign/TestMatrix.java \
+    java/foreign/TestUpcallStress.java
```

Comment:
I note that the test TestUpcallStress.java doesn't have a manual tag in the run directive ?

Reply:
Might be a mistake, TestMatrix is the manual test.

Reply:
TestUpcallStress.java was supposed to run in tier4, which is a catch-all for any test not run in lower tiers:

```
# Everything not in other tiers
tier4 = \
    / \
   -:tier1 \
   -:tier2 \
   -:tier3
```

But, it doesn't seem like TestUpcallStress is actually being run in tier4...